.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "Completed :fire:"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "scheduled",
-      "capabilityId": "ScheduledSearch",
-      "subCapability": "ScheduledSearch",
-      "version": "1.1",
-      "id": "1TuSEcCua",
-      "config": {
-        "frequency": [
-          {
-            "weekDay": 0,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          },
-          {
-            "weekDay": 1,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          },
-          {
-            "weekDay": 2,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          },
-          {
-            "weekDay": 3,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          },
-          {
-            "weekDay": 4,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          },
-          {
-            "weekDay": 5,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          },
-          {
-            "weekDay": 6,
-            "hours": [
-              13,
-              14,
-              15,
-              16,
-              17
-            ],
-            "timezoneOffset": -7
-          }
-        ],
-        "searchTerms": [
-          {
-            "name": "isIssue",
-            "parameters": {}
-          },
-          {
-            "name": "isOpen",
-            "parameters": {}
-          },
-          {
-            "name": "hasLabel",
-            "parameters": {
-              "label": "by design :art:"
-            }
-          },
-          {
-            "name": "created",
-            "parameters": {
-              "operator": "<",
-              "days": 1
-            }
-          }
-        ],
-        "taskName": "Test",
-        "actions": [
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "Test purposes only"
-            }
-          }
-        ]
-      },
-      "disabled": true
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "ReleaseAnnouncement",
-      "subCapability": "ReleaseAnnouncement",
-      "version": "1.0",
-      "id": "uFM0vTFhb",
-      "config": {
-        "taskName": "Announce a release regarding fix",
```

Comment:
This task did not get ported because release announcements aren't supported by the new service